.rubocop.yml

```diff
@@ -1,6 +1,7 @@
 AllCops:
   Exclude:
     - spec/dummy/db/schema.rb
+    - vendor/bundle/**/*
```

Comment:
Why is this?

Reply:
I did it, because of this issue https://github.com/rubocop-hq/rubocop/issues/6398